src/main/java/com/minecolonies/coremod/achievements/MineColoniesAchievement.java

```diff
@@ -13,35 +13,33 @@
  *
  * @since 0.2
  */
-public class MineColoniesAchievement extends Achievement
+public class MineColoniesAchievement
 {
-
+    //todo this will make new advancements at some point
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)